src/QuadGK.jl

```diff
@@ -16,6 +16,10 @@ using Compat
 using DataStructures, Compat.LinearAlgebra
 import Base: isless, Order.Reverse, AnyDict
 
+if !isdefined(Compat.LinearAlgebra, :rmul!)
+    const rmul! = scale!
+end
```

Comment:
Would be better to just wait for this to get tagged in Compat...

Reply:
As far as I know there's been no work on that front to make backwards-compatible versions of `mul!` et al. in Compat, so I figured this was fine for now.

Reply:
Leaving the deprecation warning for `scale!` in is a reminder that someone should work on this.

Reply:
Reverted that commit